.circleci/config.yml

```diff
@@ -6,3 +6,6 @@ workflows:
     jobs:
       - ruby-rails/lint:
           name: lint
+          executor:
+            name: ruby-rails/ruby
+            ruby-tag: '3.2.2'
```

Comment:
are you thinking of making a new release of our circleci orb for this?

Reply:
@ndushay Yep. And then removing all the manual CI ruby stuff.